zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/fetcher/Fetcher.kt

```diff
@@ -76,6 +76,15 @@ internal interface Fetcher {
   )
 }
 
+/**
+ * A manifest plus the original bytes we loaded for it. We need the original bytes for signature
+ * verification.
+ */
+class FetchedManifest(
```

Comment:
POSSIBLY this grows into the public API type we're planning,
https://github.com/cashapp/zipline/issues/566